compiler/fir/entrypoint/src/org/jetbrains/kotlin/fir/checkers/CommonDeclarationCheckers.kt

```diff
@@ -18,6 +18,7 @@ object CommonDeclarationCheckers : DeclarationCheckers() {
         FirModifierChecker,
         FirConflictsChecker,
         FirConflictingProjectionChecker,
+        FirThrowableSubclassChecker,
```

Comment:
Isn't it better to add this to `regularClassCheckers` below?

Reply:
Well, if alias `FirClassChecker` is added, then I guess there would be just `classCheckers`?